src/main/java/io/github/perrymant/moneymaker/ReportController.java

```diff
@@ -0,0 +1,15 @@
+package io.github.perrymant.moneymaker;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ReportController {
+
+    @GetMapping("/report")
```

Comment:
When Spring just sees a String return type it's not confident that what it's returning is JSON, so we should make that explicit for now (this problem goes away once we start returning POJOs).
```suggestion
    @GetMapping(value = "/report", produces = "application/json; charset=utf-8")
```